CMakeLists.txt

```diff
@@ -599,6 +613,14 @@ if(ANDROID)
     target_link_libraries(A2XT_Int INTERFACE log)
 endif()
 
+if(NINTENDO_SWITCH)
+    target_link_libraries(A2XT_Int INTERFACE nx)
+
+    # Set explicit flags for this project
+    target_compile_definitions(thextech PRIVATE
+        -D__SWITCH__ -DNO_SCREENSHOT -DFIXED_RES -DPRELOAD_LEVELS -DNO_INTPTROC)
```

Comment:
(1) Can it build without these? `-DNO_SCREENSHOT -DFIXED_RES`
(2) If files could possibly be modified when game is open, better to remove `-DPRELOAD_LEVELS` -- I think Switch filesystem is fast so it's fine to include this in the Menu Screen.

Reply:
Gonna make a shot...

Reply:
About screenshots, there is built-in screenshot functionality of Switch itself, including making short video recordings